src/lib/components/services/icons/synchronized.svelte

```diff
@@ -1,9 +1,15 @@
-<script>
+<script lang="ts">
   import { copyIcon } from "$lib/icons";
+
+  export let orange = false;
+
+  const presentation = orange
+    ? "bg-service-orange-darker text-white"
+    : "bg-service-blue text-service-blue-dark";
 </script>
 
-<div class="rounded-full bg-service-blue p-s8">
-  <div class="h-s16 w-s16 fill-current text-service-blue-dark">
+<div class={`rounded-full p-s8 ${presentation}`}>
```

Comment:
```suggestion
<div class="rounded-full p-s8 ${presentation}">
```
ou
```suggestion
<div class="rounded-full p-s8"
        class:bg-service-orange-darker={warning} class:text-white={warning} etc.
>
```

ou mieux, dans ton css
```
.warning {
@apply bg-service-orange-darker text-white
}
```
et ici 
```
     <div class="rounded-full p-s8"  class:warning>
```